### DISCUSSION TITLE
Conversion between NBT and SNBT

### DISCUSSION THREAD

**Discussion by Fsmount** [tryashtar/nbt-studio#53] [Ideas]
The idea is that the program will have a convert button that only works when you are in a file and it makes a totally new file when you click it, if you are in a NBT file and click it you will have an SNBT one and vice versa.

**Comment by tryashtar** [comment]
To save an existing file as SNBT, you can use the "save as" function. Does that do what you want or am I misunderstanding?

**Reply by Fsmount** [reply]
oh right it already exist, sorry didn't notice it thx for telling me about it
